script/installation/packages.sh

```diff
@@ -41,10 +39,10 @@ LINUX_BUILD_PACKAGES=(\
   "cmake" \
   "doxygen" \
   "git" \
-  "g++-7" \
```

Comment:
Is `g++-9` guaranteed to be picked up by something else, like `build-essential`?

Reply:
https://packages.ubuntu.com/focal/build-essential

Looks like it's >=9.2.